Description: Upstream changes introduced in version 0.5.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 hadoop-lzo (0.5.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Hortonworks Inc <jenkins@hortonworks.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- hadoop-lzo-0.5.0.orig/build.xml
+++ hadoop-lzo-0.5.0/build.xml
@@ -451,7 +451,7 @@
     </copy>
 
     <copy todir="${dist.dir}">
-	<fileset file="${build.dir}/${final.name}*.jar"/>
+	<fileset file="${build.dir}/${final.name}.jar"/>
     </copy>
 
     <exec dir="${dist.dir}" executable="sh" failonerror="true">
